### ISSUE TITLE
Add mlf-core logo to all templates?

### ISSUE THREAD

**Issue by Zethson** [mlf-core/mlf-core#59]
```
.. image:: https://user-images.githubusercontent.com/21954664/84388841-84b4cc80-abf5-11ea-83f3-b8ce8de36e25.png
    :target: https://mlf-core.com
    :alt: mlf-core logo

|
```

Could be added at the top of the README of all templates. Not quite sure actually whether this is any beneficial or not, since it may indicate that any project is a mlf-core project.

**Comment by Zethson** [comment]
Thoughts @KevinMenden @Imipenem @Emiller88 ?

**Comment by Imipenem** [comment]
I mean, this is cool for projects in the mlf-core orga but otherwise I can't really see any advantages + some users might not want this at all (although one could make this opt-in).

**Comment by KevinMenden** [comment]
Agree, alternatively the logo could be shown in smaller, at the bottom of the page for instance, as an acknowledgement

**Comment by Zethson** [comment]
Cool, thanks for the feedback.
I like @KevinMenden suggestion and might implement that.

**Comment by Zethson** [comment]
We should then also add the preprint to cite to the templates README at the very bottom when it is out.
